docs/content/concepts/assets/asset-materializations.mdx

```diff
@@ -330,3 +332,6 @@ src="/images/concepts/assets/asset-lineage.png"
 width={1607}
 height={1031}
 />
+
+## Software-defined assets vs Asset Materializations
+In this section we'll go over some differences between software-defined assets and asset materializations and the different use cases for each.
```

Comment:
realizing i don't know what to add in this section. i'm pretty fuzzy on the intricacies or details of why you might use `@asset` vs an asset materialization. they seem similar to me in terms of effect. the main difference seems to be more from an organizational standpoint or a general approach to how you handle data assets

Reply:
The biggest difference is that:
* With software-defined assets, the assets are known at definition time - i.e. when you load the code into Dagit, even before you run it, you know what assets are going to be materialized. Dagit can render the asset graph just by looking at the asset definitions. And the asset page can include a "Materialize" button, because we know what op to run to materialize an asset.
* AssetMaterializations, on the other hand, are logged at run time. I.e. you find out what assets an op materialized when you run the op. They're more about describing what happened than about describing what you intend to happen.

Reply:
ok! lmk what you think of this blurb